web/vtadmin/src/components/routes/tablet/FullStatus.tsx

```diff
@@ -1,15 +1,29 @@
 import React from 'react';
 import { Link } from 'react-router-dom';
 import { useGetFullStatus } from '../../../hooks/api';
-import { vtadmin } from '../../../proto/vtadmin';
+import { topodata, vtadmin } from '../../../proto/vtadmin';
 import { formatAlias } from '../../../util/tablets';
 import { Code } from '../../Code';
 import style from './Tablet.module.scss';
+import { isNil } from 'lodash-es';
 
 interface Props {
     tablet: vtadmin.Tablet;
 }
 
+function stateReplacer(key: any, val: any) {
+    if (key === 'io_state' || key === 'sql_state') {
+        if (val === 3) {
+            return 'Replication Running';
+        } else if (val === 2) {
+            return 'Replication Connected';
+        } else if (val === 1) {
+            return 'Replication Stopped';
+        }
```

Comment:
since this is already in the context of "IO Thread" or "SQL Thread" (in the ui, i mean), is "replication" maybe redundant here? what i am thinking is we could maybe just do:

```json
{
  "io_thread": "{running|connected|stopped}",
  "sql_thread": "{running|connected|stopped}"
}
```

Reply:
Agree with @ajm188. Further, assuming that the source for this enum is https://github.com/vitessio/vitess/blob/dcd1d3b2e4f6d4a4f71979a6403ca889df3bb8c7/go/mysql/constants.go#L739-L744 
2 corresponds to `Connecting` not `Connected`. And we might want to complete the set and represent `0` as `Unknown`.

Reply:
Fixed the strings. I didn't add the Unknown field for 0 because for the primary tablet, the IO_State and SQL_state aren't populated, which then defaults to being 0. We don't want to show Unknown state there.